src/index.ts

```diff
@@ -34,6 +36,13 @@ export async function run(): Promise<void> {
     // remove white spaces (if any) in archs input
     archs = archs.replace(/\s+/g, "");
 
+    // To solve https://github.com/redhat-actions/buildah-build/issues/45
+    const homedir = os.homedir();
+    const storageFilePaths: string[] = [ path.join(homedir, ".config/containers/storage.conf"),
```

Comment:
~/.config usually comes from `XDG_CONFIG_HOME`. Use that if it's set, or default to ~/.config

https://wiki.archlinux.org/index.php/XDG_Base_Directory#User_directories